Packs/illuminate/Integrations/Analyst1/Analyst1.py

```diff
@@ -350,16 +451,293 @@ def url_command(client: Client, args: dict) -> List[EnrichmentOutput]:
+        indicators_added = a1_tasking_array_from_indicators(simplified_data['indicatorsAdded'])
+        del simplified_data['indicatorsAdded']
+
+    indicators_removed: dict = {}
+    if 'indicatorsRemoved' in simplified_data:
+        indicators_removed = a1_tasking_array_from_indicators(simplified_data['indicatorsRemoved'])
+        del simplified_data['indicatorsRemoved']
+
+    rules_added: dict = {}
+    if 'rulesAdded' in simplified_data:
+        rules_added = a1_tasking_array_from_rules(simplified_data['rulesAdded'])
+        del simplified_data['rulesAdded']
+
+    rules_removed: dict = {}
+    if 'rulesRemoved' in simplified_data:
+        rules_removed = a1_tasking_array_from_rules(simplified_data['rulesRemoved'])
+        del simplified_data['rulesRemoved']
+
+    command_results_list: List[CommandResults] = []
+
+    command_results = CommandResults(
+        outputs_prefix='Analyst1.SensorTaskings',
+        outputs=simplified_data,
+        raw_response=raw_data
+    )
+    return_results(command_results)
+    command_results_list.append(command_results)
+
+    command_results = CommandResults(
+        outputs_prefix='Analyst1.SensorTaskings.IndicatorsAdded',
+        outputs_key_field='id',
+        outputs=indicators_added
+    )
+    return_results(command_results)
+    command_results_list.append(command_results)
+
+    command_results = CommandResults(
+        outputs_prefix='Analyst1.SensorTaskings.IndicatorsRemoved',
+        outputs_key_field='id',
+        outputs=indicators_removed
+    )
+    return_results(command_results)
+    command_results_list.append(command_results)
+
+    command_results = CommandResults(
+        outputs_prefix='Analyst1.SensorTaskings.RulesAdded',
+        outputs_key_field='id',
+        outputs=rules_added
+    )
+    return_results(command_results)
+    command_results_list.append(command_results)
+
+    command_results = CommandResults(
+        outputs_prefix='Analyst1.SensorTaskings.RulesRemoved',
+        outputs_key_field='id',
+        outputs=rules_removed
+    )
+    return_results(command_results)
+    command_results_list.append(command_results)
+
+    return command_results_list
+
+
+def analyst1_get_sensor_config_command(client: Client, args):
+    sensor_id = args.get('sensor_id')
+    raw_data = client.get_sensor_config(sensor_id)
+    warRoomEntry = fileResult('sensor' + str(sensor_id) + 'Config.txt', raw_data)
+    outputOptions = {'warRoomEntry': warRoomEntry, 'config_text': raw_data}
+
+    command_results = CommandResults(
+        outputs_prefix='Analyst1.SensorTaskings.ConfigFile',
+        outputs=outputOptions
+    )
+    return_results(command_results)
+    return command_results
+
+
 ''' EXECUTION '''
 
 
 def main():
     commands = {
-        'domain': domain_command,
-        'email': email_command,
-        'file': file_command,
-        'ip': ip_command,
-        'url': url_command,
+        'analyst1-enrich-domain': domain_command,
```

Comment:
Why were the names of the commands changed?
This will break backwards compatibility for customers.
The reason the commands were called that, is because they are Reputation Commands, you can see [here](https://xsoar.pan.dev/docs/integrations/generic-commands-reputation).

Reply:
Thank you. Will revert. It was done for consistency & in some coordination with current users awareness. But that was a personal oversight in my process and this will go back at next commit.